Dockerfile

```diff
@@ -60,9 +72,9 @@ RUN find ${PY_PATH} -type f -a -name '*.pyo' -exec rm -rf '{}' +
 #######################
 FROM python:3.8-alpine AS runner
 
-RUN apk add --no-cache tini
+RUN apk -U upgrade --no-cache
+RUN apk add --no-cache tini gmp libstdc++ libgcc
 
-COPY --from=rust-builder ["/usr/lib/libstdc++.so.6", "/usr/lib/libgcc_s.so.1", "/usr/lib/libgmp.so.10", "/usr/lib/" ]
```

Comment:
the /usr/lib looks extra and dangerious thing to copy... Unless it actually means the destination? Weird syntax though. If it's the destination then the three packages should have exactly the same end result.

I did try to test this by running `python -m pytest` successfully when py stuff was carried over to the final image.

Reply:
Yes, "/usr/lib/" is the destination: https://docs.docker.com/engine/reference/builder/#copy

It's a strange syntax indeed (basically behaves like `cp`, but I agree that it could have been made more explicit here).

Reply:
Then this should be all right.